### PR TITLE
Datadog/agent failing win2008/ma 2122

### PR DESCRIPTION
**Pull request by bhargavapratik**

### CODE REVIEW

**Comment by bhargavapratik** [comment]
@chandramishra @kameshsg  please check and validate cf_init  , I have resolved conflict in it.

File: managed-os/build.f/particles-managed-os/particles/conditions/isNeedToSkipSigAuthForDataDog.hbs [hunk 1]
```diff
@@ -0,0 +1 @@
+"Fn::Or": [{"Fn::Equals":[{"Ref": "OSName"},"Windows Server 2008 SP2 64-bit"]},{"Condition": "is32BitOs"}]
```
Comment (RajkumarShivage) [inline managed-os/build.f/particles-managed-os/particles/conditions/isNeedToSkipSigAuthForDataDog.hbs]: what is "is32BitOs"?

Comment (RajkumarShivage) [reply managed-os/build.f/particles-managed-os/particles/conditions/isNeedToSkipSigAuthForDataDog.hbs]: How are we handling Win2k8 32-bit and 64-bit?

Comment (bhargavapratik) [reply managed-os/build.f/particles-managed-os/particles/conditions/isNeedToSkipSigAuthForDataDog.hbs]: I think is32BitOs condition check it.

Comment (kameshsg) [reply managed-os/build.f/particles-managed-os/particles/conditions/isNeedToSkipSigAuthForDataDog.hbs]: @RajkumarShivage  I am using "or" condition for Win2008 64bit or is32BitOs check. I have referred -  http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-conditions.html#intrinsic-function-reference-conditions-or for putting this condition

Comment (kameshsg) [reply managed-os/build.f/particles-managed-os/particles/conditions/isNeedToSkipSigAuthForDataDog.hbs]: @RajkumarShivage  **is32BitOs** condition checks its Win2008 32bit os. (Its previously added condition, I am just reusing along with new condition ie. win 2008 64bit check)

**Pull request merged**